mreg/models.py

```diff
@@ -786,3 +787,25 @@ class History(models.Model):
 
     def __str__(self):
         return f'{self.name}, {self.model}, {self.action}, {self.timestamp}'
+
+
+class BACnetID(models.Model):
+    id = models.IntegerField(primary_key=True, validators=[validate_BACnetID])
+    host = models.OneToOneField(Host, on_delete=models.CASCADE, related_name='bacnetid')
+
+    class Meta:
+        db_table = 'bacnetid'
+
+    @property
+    def hostname(self):
+        return self.host.name
+
+    @staticmethod
+    def first_unused_id() -> int:
+        j = 0
+        for i in BACnetID.objects.values_list('id', flat=True).order_by('id'):
+            if i==j:
+                j = j+1
```

Comment:
`j += 1`

Reply:
Fixed just now in 894fb5480f3558764890c1c388e78da7a7cdb3d3